flang/test/Driver/fveclib-codegen.f90

```diff
@@ -0,0 +1,13 @@
+! test that -fveclib= is passed to the backend
+! -target aarch64 so that ArmPL is available
+! RUN: %flang -S -target aarch64-unknown-linux-gnu -mcpu=neoverse-v1 -Ofast -fveclib=ArmPL -o - %s | FileCheck %s
+
```

Comment:
Will this work on all targets? Or only on AArch64?

Reply:
-fveclib should work on all targets which clang supports (I have only tested aarch64). ArmPL is only available on aarch64 (and this is enforced in the compiler driver) so I added the target here. I used ArmPL for the test because I know what the right result looks like for ArmPL, but not for other libraries.

Reply:
I was asking whether this test needs to be restricted to only work on a system where Arm target is enabled.